### PR TITLE
#486 - Refactoring project structure (3.3.0) 

### PR DESCRIPTION
**Pull request by astier**
- Replace Stringbuffer with StringBuilder
- Remove unnecessary unboxing
- Remove unnecessary for-loop

### CODE REVIEW

File: webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/AnnotationSchemaServiceImpl.java [hunk 1]
```diff
@@ -803,8 +803,8 @@ private AnnotationFeature createFeature(String aName, String aUiname, Project aP
                 .createQuery("FROM Tag WHERE tagSet = :tagSet ORDER BY name ASC", Tag.class)
                 .setParameter("tagSet", aTagSet).getResultList();
         // FIXME ?!? This loop appears to make absolutely not sense!
```
Comment (astier) [inline webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/AnnotationSchemaServiceImpl.java]: I think this loop can actually be removed.

Comment (astier) [inline webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/AnnotationSchemaServiceImpl.java]: Whoops. The loop below which you can not see.

Comment (reckart) [inline webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/AnnotationSchemaServiceImpl.java]: Probably yes...

**Comment by astier** [comment]
I changed now for this PR the commit message by squashing all the commits. I also removed the mentioned unnecessary for-loop.

**Comment by astier** [comment]
Since I replaced StringBuffer with StringBuilder can it cause any problems because StringBuilder is not Synchronized?

**Comment by reckart** [comment]
> Since I replaced StringBuffer with StringBuilder can it cause any problems because StringBuilder is not Synchronized?

I don't think so.

**Pull request merged**